spec/features/users/user_can_comment_on_question_spec.rb

```diff
@@ -3,6 +3,7 @@
 describe 'when a user visits a question show page' do
```

Comment:
Good job adding the roles to users. Katy has found a way to build that functionality out with Fabricator and will let us know when it makes it's way up.